lib/node_modules/@stdlib/blas/ext/base/ssumpw/examples/index.js

```diff
@@ -18,18 +18,11 @@
 
 'use strict';
 
-var randu = require( '@stdlib/random/base/randu' );
-var round = require( '@stdlib/math/base/special/round' );
-var Float32Array = require( '@stdlib/array/float32' );
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' ).factory;
+var filledarrayBy = require( '@stdlib/array/filled-by' );
 var ssumpw = require( './../lib' );
 
-var x;
-var i;
-
-x = new Float32Array( 10 );
-for ( i = 0; i < x.length; i++ ) {
-	x[ i ] = round( randu()*100.0 );
-}
+var x = filledarrayBy(10, 'float32', discreteUniform(-100.0, 100.0));
```

Comment:
```suggestion
var x = filledarrayBy( 10, 'float32', discreteUniform( -100.0, 100.0 ) );
```

Spacing here and elsewhere.